### DISCUSSION TITLE
Adding Hyperlink on Notification Tab

### DISCUSSION THREAD

**Discussion by SanchanaMohankumar** [microsoft/Microsoft-Fabric-workload-development-sample#26] [Q&A] [answered]
Scenario: Currently, in the attached image below, we see an explore button on the Transform completed notification tab. We have to provide a hyperlink for it. However, we do not see any references in the given repository. We also tried implementing it as per the attached Microsoft documentation link below, but it is still not working. Please do provide your input.
Link - https://learn.microsoft.com/en-us/javascript/api/@ms-fabric/workload-client/opennotificationbutton?view=fabric-extensibility-js-latest 
<img width="657" alt="image" src="https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/assets/175164325/27d67b5d-f2a3-4cb6-98a8-9e287c533b84">

**Comment by anitamayorov** [comment, marked as answer]
Hi @SanchanaMohankumar,

Link button type is a deprecated feature as stated in documentation, and will not be supported in the future, therefore notifications will contain text only at this stage.
